### PR TITLE
Add data driven summaries to the data portal

### PR DESCRIPTION
**Pull request by michaelglenister**
Issue: https://trello.com/c/2qybjQ9z/604-make-data-summaries-data-driven

### CODE REVIEW

**Review by paulmwatson — APPROVED** [review]

**Review by paulmwatson — APPROVED** [review]

File: municipal_finance/summarise_data.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+def summarise():
+    min_year = 3000
+    max_year = 1000
+    count_facts = 0
+
+    for table in FACT_TABLES:
+        min_year = get_min(min_year, table)
+        max_year = get_max(max_year, table)
+
+    count_years = max_year - min_year
```
Comment (jbothma) [inline municipal_finance/summarise_data.py:83]: is this an off by one error? looks like when I upload a file with a single year of data, it says I have zero years of data

File: municipal_finance/summarise_data.py [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    Summary.objects.update_or_create(
+        type='years',
+        defaults={
+            'content': f'{{"count":{count_years}, "min":{min_year}, "max":{max_year}}}'}
+    )
+
+    total = Geography.objects.all().count()
+    metros = Geography.objects.filter(category='A').count()
+    munis = Geography.objects.filter(category='B').count()
+    districts = Geography.objects.filter(category='C').count()
```
Comment (jbothma) [inline municipal_finance/summarise_data.py:93]: this needs to exclude disestablished municipalities https://municipaldata.treasury.gov.za/api/cubes/demarcation_changes/facts

Comment (jbothma) [reply municipal_finance/summarise_data.py:93]: otherwise you'll be over-counting

**Review by jbothma — CHANGES_REQUESTED** [review]
great! Just a couple of smallish changes needed I think

Could you also add a test user to demodata and document a la https://github.com/OpenUpSA/wazimap-ng/blob/staging/demodata.json#L145

File: municipal_finance/summarise_data.py (hunk 1 above)
Comment (michaelglenister) [reply municipal_finance/summarise_data.py]: Changed to count a list of years and get min max from that without magic numbers

File: municipal_finance/summarise_data.py (hunk 2 above)
Comment (michaelglenister) [reply municipal_finance/summarise_data.py]: Discussed in standup - added a note making it clear that disestablished municipalities are included in the data

Comment (jbothma) [reply municipal_finance/summarise_data.py]: nice - I don't think `sup` is the right thing - I think it needs space below the big text, and a matching *. The size of the text is fine, so just style it another way to get smaller text and a bit more space

Comment (michaelglenister) [reply municipal_finance/summarise_data.py]: Changed to <p>, added matching asterisk and extra space

**Review by jbothma — APPROVED** [review]

**Pull request merged**